Makefile

```diff
@@ -1,4 +1,6 @@
 ASYNCIFY_DEFAULT_STACK_SIZE?=2097152
+USE_STACK_POOLING?=0
```

Comment:
Is this supposed to eventually control stack pooling for both asyncify and wasmfx simultaneously? If not, both of these should be prefixed with `ASYNCIFY_`

Reply:
I'd like to control both implementations, eventually. For that to happen, we need to expose the WasmFX config in the C API of Wasmtime.